doc/integration.md

```diff
@@ -126,6 +129,105 @@ Note the following:
 
+##Performing purchases
+The aim of this SDK is to allow users to make purchases using your application, backed by Klarna as a payment method. However, the SDK does not offer any direct methods for performing purchases as this will expose your application's private Klarna credentials. Instead, applications using the SDK are expected to work in concert with an application backend, which will perform the actual purchase requests.
+
+In this section, we will see how to communicate with such a backend and for that purpose we supply a sample backend that you can find [here](https://github.com/klarna/sample-ondemand-backend). Reading the sample backend's documentation will allow you to fully grasp how an application using this SDK is expected to perform purchases, and you are encouraged to take a look if things become too unclear.
+
+###Signing requests
+While you can, and almost certainly will, communicate with your application's backend in a way that is different from the very simplistic approach we present here, one thing you will always have to do is sign your purchase requests. This will significantly increase your user's security while buying and the SDK makes this task incredibly easy.
+
+Let us say a user wants to make a purchase for a total of 40.50 Euros. All that's necessary to generate the relevant signature is to import the origin proof class:
+
+```java
+import com.klarna.ondemand.OriginProof;
+```
+
+which will allow you to construct an object as seen below:
+
+```java
+OriginProof originProof = new OriginProof(4050, "EUR", getUserToken(), getApplicationContext());
+```
+
+Assume `getUserToken()` returns the user's token as received during registration. Note that the constructor expects the purchase amount to be supplied in cents. You can find the constructor's full documentation [here](http://klarna.github.io/klarna-on-demand-android/com/klarna/ondemand/OriginProof.html#OriginProof-int-java.lang.String-java.lang.String-android.content.Context-).
+
+###Purchase example
+We now know how to generate the signature required for a purchase to go through. Let us see how to send it, along with other required information, to the sample backend.
+
+You will most likely have a "buy" button somewhere in your application. The code below shows how such a button might be implemented in your application's controller:
+
+```java
+private void buyTicket() {
+  // create an origin proof, as seen in the previous section (notice this is not the exact same call)
+  final OriginProof originProof = new OriginProof(9900, "SEK", getUserToken(), getApplicationContext());
```

Comment:
It's actually a call with different parameters on purpose, since I want it to resemble what we have going on in the sample app. Maybe I'll change the 4050 to 9900 when I handle all the feedback we got.